src/PayClient.js

```diff
@@ -0,0 +1,212 @@
+const Promise = require('bluebird');
+const req = require('requestretry');
```

Comment:
One problem we've seen with FRS is the client retrying a donation after getting an ETIMEDOUT or something similar on the first try (usually due to the payment processor itself getting bogged down), and ending up being double-charged. Seems like automatically retrying any request someone sends to Pay through this client might open the door to similar errors.

Reply:
I talked to Jon about this.  There was a reason he used requestretry ages ago but we don't think it applies any more.  I'll switch this to just request.